client/whispers/whisper-reducer.ts

```diff
@@ -68,9 +68,9 @@ function updateMessages(
 
 export default immerKeyedReducer(DEFAULT_STATE, {
   ['@whispers/getWhisperSessions'](state, action) {
-    state.sessions = new Set(action.payload)
+    state.sessions = new Set(action.payload.sessions)
 
-    for (const session of action.payload) {
+    for (const session of action.payload.sessions) {
```

Comment:
might need to merge these instead of assuming they are completely unset?

Reply:
You mean something like this?
```ts
for (const session of action.payload.sessions) {
  if (state.byId.has(session)) {
    continue
  }

  state.byId.set(session, defaultWhisperSession(session))
}
```

Reply:
sure